mscore/scoreview.cpp

```diff
@@ -92,6 +92,9 @@
 #include "libmscore/textline.h"
 #include "libmscore/shape.h"
 
+#include <avsomr/avsomr.hpp>
+#include <avsomr/avsomrdrawer.hpp>
```

Comment:
I guess this should rather be double quotes, not angle brackets

Reply:
Usually, I will become angled quotes when I do an inclusion from another library, thereby showing that this is an external implementation (relative to the current library), and this implementation may change. If it is customary to put double quotes everywhere, then I will fix it.

Reply:
> During development I'm comfortable with my current style (this is the default style in the QtCreator)

There's an XML you can import in QtCreator to fix the coding style for you, see https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules#Coding-style